crates/driver/src/util/percent.rs

```diff
@@ -0,0 +1,27 @@
+use thiserror::Error;
+
+/// A percentage value. The value is guaranteed to be in the range [0f64,
```

Comment:
I think in other parts of the codebase, percentage types are interpreted as a factor (e.g. they range is between 0 and 1), e.g. https://github.com/cowprotocol/services/blob/148a922e329229ddeab03901624c3a73ef89b2d3/crates/shared/src/arguments.rs#L514-L518. 

Not a big difference, but I'd like to be consistent so we avoid accidental misconfiguration. I can see somebody setting `solve_competition_time_buffer_percent` to 0.2 and expecting it to be 20% (like our other percentage values). The system would run without complaining, however - if we make it a factor - misconfiguring the value 20 would be caught at start-up time.

Reply:
Fixed.